core/settings.py

```diff
@@ -114,25 +114,25 @@
     # }
 
 
+    'default': {
+    'ENGINE': 'django.db.backends.postgresql',
+    'NAME': 'neondb',
+    'USER': 'dimeji0518',
+    'PASSWORD': 'cy8vHl7ASYTj',
+    'HOST': 'ep-round-truth-56973996.us-east-2.aws.neon.tech',
+    'PORT': '5432',
+    'OPTIONS': {'sslmode': 'require'},
+    }
+
```

Comment:
Do not expose secrets and push them on Github.
This is a dangerous bad practice.
Use env variables.

Reply:
ok done thanks